Sources/Vapor/Commands/ServeCommand.swift

```diff
@@ -44,8 +47,13 @@ public final class ServeCommand: Command {
         let port = signature.port
             // 0.0.0.0:8080, :8080, parse port
             ?? signature.bind?.split(separator: ":").last.flatMap(String.init).flatMap(Int.init)
-
-        try context.application.server.start(hostname: hostname, port: port)
+        let socketPath = signature.socketPath // /tmp/vapor.socket
+        
+        if let socketPath = socketPath, !socketPath.isEmpty {
```

Comment:
The socket path should never be empty since this is an `@Option` and not a flag. We can assert / precondition this.

Reply:
So this should be enough then?
```suggestion
        if let socketPath = socketPath {
```